File: 2014-10-CSharp-Fundamentals-1/2. Primitive Data Types and Variables/IntegerTypes/IntegerTypes.cs
using System;

class IntegerTypes
{
    static void Main()
    {
        byte centuries = 20;    // Usually a small number
        ushort years = 2000;
        uint days = 730480;
        ulong hours = 17531520; // May be a very big number

        Console.WriteLine(
			"{0} centuries is {1} years, or {2} days, or {3} hours.",
			centuries, years, days, hours);
    }
}

File: 2015-10-Java-Fundamentals-1/02. Data-types-and-variables/demos/IntegerTypes.java
import java.text.MessageFormat;

public class IntegerTypes {
    public static void main(String[] args) {
        byte centuries = 20;    // Usually a small number
        short years = 2000;
        int days = 730480;
        long hours = 17531520; // May be a very big number

        writeLine(
            "{0} centuries is {1} years, or {2} days, or {3} hours.",
            centuries, years, days, hours);
    }

    public static void writeLine(String format, Object... obj) {
        MessageFormat form = new MessageFormat(format);
        System.out.println(form.format(obj));
    }
}
